assets/src/edit-story/components/richText/karma/itf.karma.js

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma';
+import { useStory } from '../../../app/story';
+
+describe('Background Overlay Panel', () => {
+  let fixture;
+  let bgId, textId;
+
+  beforeEach(async () => {
+    fixture = new Fixture();
+    await fixture.render();
+
+    // Add new text
+    const addText = fixture.screen.getByLabelText(/Add new text element/i);
+    await fixture.events.click(addText);
+
+    const {
+      state: {
+        currentPage: { elements },
+      },
+    } = await fixture.renderHook(() => useStory());
+    bgId = elements[0].id;
+    textId = elements[1].id;
+  });
+
+  afterEach(() => {
+    fixture.restore();
+  });
+
+  it('should have the correct initial text', async () => {
+    expect(await getTextContent()).toBe(
+      '<span style="font-weight: 400">Fill in some text</span>'
```

Comment:
So it's actually a little weird that it comes with font-weight: 400, as it's not necessary nor will it be output by the rich text editor (as 400 is default).

Reply:
Hmm, the `DEFAULT_PRESET` includes the default font-weight, should we just remove it?

Reply:
Yes it should. Will do.